### DISCUSSION TITLE
AAVE testnet UI issue with withdrawing my complete deposited amount, wanted to understand what is happening here, for knowledge

### DISCUSSION THREAD

**Discussion by WeTheArtist** [smartcontractkit/full-blockchain-solidity-course-py#645] [Q&A]
I wanted to ask this question to help my understanding, I am not sure if this is only because of using the testnet or if it is something normal.
So I deposited 1 ETH on the AAVE testnet and got 1 aETH as well, added to wallet. 
Now when I am trying to withdraw it allows me to withdraw only 0.5 ETH, I cannot withdraw that complete as well because it gives an error saying that there  is insufficient funds.
So I tried to withdraw lesser amounts, like 0,2 and it worked. So now while withdrawing the remaining 0.3 it says the following error (picture).
Is it because someone has borrowed this ETH and there is less available on AAVE that I cannot withdraw it? 
I thought that you can withdraw your lended amount whenever you want.

![image](https://user-images.githubusercontent.com/79281462/147507311-101a8db7-0e35-4408-bb73-f421c0d799e6.png)

**Comment by WeTheArtist** [comment]
Cuurent amount available:
![image](https://user-images.githubusercontent.com/79281462/147507406-d5f74e7d-a724-4d76-9964-c5cdda9600f9.png)

after clicking on withdraw:
![image](https://user-images.githubusercontent.com/79281462/147507428-5562161b-a87c-4251-95e6-09b804781dce.png)

error after choosing max amount and continue.:
![image](https://user-images.githubusercontent.com/79281462/147507466-1ce52b05-6ccb-49e1-9740-6d417a223fff.png)

I do have sufficient ETH and aETH in wallet:
![image](https://user-images.githubusercontent.com/79281462/147507565-18da06a1-8710-4c3b-8339-12c7500bd159.png)


any explanation to understand what is going on here is very appreciated. I tried to search on google but there was no real answer.

**Reply by cromewar** [reply]
Yes you are right, when you do this transactions you set your tokens as liquidity for other people, as the same you get other tokens in exchange, so sometimes you should wait a little to retrieve them all, also take in consideration aave is not meant to be used like this (borrowing and retrieving all the same day), so the token you give also gets locked and liquidity for some time. 
Also withdrawing the complete amount does not work as you must pay gas fees directly on aave, so you must always  have a little bit more tokens.
